### PR TITLE
remove duplicate line and add a new common usage.

### PR DESCRIPTION
**Pull request by matthewgao**
remove duplicate lines and add a new common usage of '-avz'

### CODE REVIEW

**Comment by igorshubovych** [comment]
Could you please think about making 2nd example shorter?

**Comment by matthewgao** [comment]
you mean this one?

transfer file from remote host to local
`rsync {{remote_host_name}}:{{remote_file_location}} {{local_file_location}}`

**Comment by igorshubovych** [comment]
i mean this one

- transfer file in archive and compress mode, it ensures symbolic link, device, attr, permissions, are preserved, and reduce the size.

The text is too long.

**Comment by matthewgao** [comment]
OK

**Comment by waldyrious** [comment]
Thanks, great contribution! A few observations:

- perhaps write "compressed (zipped)" to make the connection to the option name clearer. The "and reduces the size" is unnecessary as that's exactly what compressed means.
- What does the -v do? It's not clear.

**Comment by matthewgao** [comment]
Reasonable, will update.

-v actually means verbose.

**Comment by waldyrious** [comment]
Ok so I'd suggest leaving -v out in that example. The goal is to introduce options gradually and allow beginners to infer the meaning of the examples, not just provide handy commands to copy-paste.

So I'd phrase the example description as follows:
"transfer file in archive (to preserve attributes) and compressed (zipped) mode"

**Comment by matthewgao** [comment]
Cool!

File: pages/common/rsync.md [hunk 1]
```diff
@@ -4,7 +4,7 @@
 > Does not allow transfer between two remote hosts
 > Can transfer single files or files matched by pattern
 
-- transfer file from local to remote host
+- transfer file from local to remote host, it supports regex.
```
Comment (waldyrious) [inline pages/common/rsync.md]: can you replace ", it supports regex" with "(Note: filename supports regex)"? The current way doesn't form a proper sentence.

File: pages/common/rsync.md [hunk 2]
```diff
@@ -4,7 +4,7 @@
...
 
 `rsync {{path_to_file}} {{remote_host_name}}:{{remote_host_location}}`
```
Comment (waldyrious) [inline pages/common/rsync.md]: how about `{{path/to/file}}`? I think it'd drive the point home better.

**Comment by waldyrious** [comment]
Just two more details. Sorry for nitpicking :)

**Comment by matthewgao** [comment]
No problem, that's what we are pursuing. :)

**Comment by waldyrious** [comment]
Awesome, thanks for the quick reactions and your patience! Looks good to merge :) :+1:

**Comment by felixonmars** [comment]
:shipit:

File: pages/common/rsync.md [hunk 3]
```diff
@@ -4,21 +4,21 @@
 > Does not allow transfer between two remote hosts
 > Can transfer single files or files matched by pattern
 
-- transfer file from local to remote host
+- transfer file from local to remote host, filename supports regex.
```
Comment (ostera) [inline pages/common/rsync.md]: Is this true? Or is it a glob expansion from the shell that turns into a list of files?

**Comment by igorshubovych** [comment]
It is almost good to go. I guess we can continue fix it in a later pull requests.

@matthewgao could you please squash the commits? Let me know if you need help with it.

**Comment by matthewgao** [comment]
How can I squash the commits?

**Comment by igorshubovych** [comment]
https://www.youtube.com/watch?v=qh9KtjfjzCU

**Comment by rubenvereecken** [comment]
Die you old PR die.

Thanks for the contribution @matthewgao !

**Comment by waldyrious** [comment]
Too bad @ostera's question was left unanswered. I hate leaving loose ends like that :/

**Comment by rubenvereecken** [comment]
Ok don't tell anyone but I chucked out that particular line because I assume they're shell globs. Wouldn't mind adding in later (yes I know that won't happen), but let's keep fresh and going.

**Comment by waldyrious** [comment]
@rubenvereecken roger that. Anyway, maybe @matthewgao will comment later on so we can lay the issue to rest :)

**Pull request merged**